tsconfig.eslint.json

```diff
@@ -6,7 +6,8 @@
     "./config/**/*.ts",
     "./tests/**/*.ts",
     "./packages/**/*.ts",
-    "./packages/**/*.tsx"
+    "./packages/**/*.tsx",
+    "faker.d.ts"
```

Comment:
This line can be removed if you move the d.ts file into the config directory per my earlier comment